adhocracy4/comments/serializers.py

```diff
@@ -12,6 +12,7 @@ class CommentSerializer(serializers.ModelSerializer):
     is_deleted = serializers.SerializerMethodField()
     ratings = serializers.SerializerMethodField()
     is_moderator = serializers.SerializerMethodField()
+    is_blocked = serializers.SerializerMethodField()
```

Comment:
SerializerMethodField is only necessary when you dont want to serialize the actual value of the field, but do sth with it before, like for user_name, return 'unknown user' if the comment has been deleted. Since for is_blocked, we only want to serialize the actual value True/False, you only need to add it to the list of fields in Meta Class or, since this is the way it is done here, not add it to the exclude list. So in short: remove this line and the get_is_blocked method and it will work fine (same for async serializer).

Reply:
But in fact, I would not even add it to this serializer as we only use is_blocked in Kosmo so far which uses the async serializer. Do you agree @fuzzylogic2000 ? So: in this serializer please add it to the exclude list and in the async one, what I wrote above is valid :) Sorry, I hope this is not too confusing..

Reply:
No, everything makes perfect sense, thank you!! Just one clarification question: so you mean I should add it **only** to the exclude list for the regular comments/serializer.py and don't exclude it in the comments_async/serializer?

Reply:
No, that's exactly what @Rineee meant.
There are two ways to define what is in the serializer. One is naming the fields that should be there and the other one excluding what shouldn't. In the comment serializers we use the latter.
So, to use the is_blocked, you do not exclude it and thus it is shown. But to only show and use it in kosmo and not add it to the "old" comments, you do exclude it in the comment serializer.